package.json

```diff
@@ -54,7 +54,7 @@
     "test:unit": "istanbul cover tape 'test/**/*.test.js'",
     "build": "run-p build:*",
     "build:js": "browserify client/main.js -o public/bundle.js",
-    "build:css": "node-sass client/styles/main.scss public/bundle.css --include-path node_modules/foundation-sites/scss --include-path node_modules/slick-carousel/slick",
+    "build:css": "node-sass client/styles/main.scss public/bundle.css --include-path node_modules/foundation-sites/scss --include-path node_modules/slick-carousel/slick  && postcss --u autoprefixer -b 'last 2 versions, ie >= 9' -i public/bundle.css -o public/bundle.css",
```

Comment:
I think it'll be best to split this into two scripts, leaving "build:css" to run them, something like:

```json
{
  "build:css": "run-s build:css:*",
  "build:css:sass": "...",
  "build:css:prefix": "...",
}
```